shopservice/src/main/java/moviecatalog/shopservice/controller/ShopController.java

```diff
@@ -45,6 +45,7 @@ public Order createMovieOrder(final @Valid @RequestBody OrderDto dto) {
 
     @GetMapping("/user/{userId}")
     public UserOrders getAllOrdersForUser(final @PathVariable Long userId) {
+        LOGGER.info("getting user orders request");
```

Comment:
nit: tohle by pak jinde klidně mohlo mít třeba debug nebo trace level. Protože samo o sobě ti to moc neřekne v logu.
Tak to jen tak bokem.

Reply:
vytvoreno issue #206